governance-exercises/JanSky Polling Exercise 4 - Forum Post.md

```diff
@@ -0,0 +1,20 @@
+# Forum Response
+
+Author: JanSky  
+Date: Wednesday 9 August 2023
+
+---
+
+This situation requires an immediate response. A couple of recommendations that should happen promptly:
+
+**Emergency Poll on Fix A**: I will be initiating an emergency poll presently to see if there is community support for Fix A. **This poll will only last 12 hours, @Aligned Delegates please vote as soon as you are ready and able.** While [MIP24](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) states that a poll might not be strictly necessary in emergencies, I am recommending it given the difference of opinion between Roundtable and WeCodeCrypto.
+
+**Emergency Governance Call**: I am organizing a governance call with Roundtable Develop, WeCodeCrypto, and Imogen. @Ecosystem Facilitator should also attend if they are able (please message me on Discord for information, you have a message there in your inbox). I will summarize and release notes from this call to the community once this Emergency Response stops.
+
+
+Imogen's expertise in Executive spell preparation should be respected. As Protocol Scope Facilitator, it is their call what goes in the emergency executive. I hope the poll and emergency call coordination aid their decision, but please note Imogen is not bound by the results. Should large support for Fix A arise during the emergency poll, I will advise initiating preparations for its implementation before the polling period ends.
+
+### Note on Decorum
+
+We are using this thread for public communications about the progress of this Emergency Response. I respect that tensions may be high because of differences in opinion over the best course of action. Please keep politizing the issue out of this thread. I will delete off-topic or inflammatory comments will and move the debate to separate threads. 
```

Comment:
Yeah, I covered this in the comment in the summary, but I think you are pulling the trigger one step too early here. If Genevieve or Henrik respond further, then I think this is justified. 

Totally worth addressing the need for decorum and mutual respect after the emergency has passed, that is what postmortems are for. However, right now the goal is to keep everyone focused on the emergency. Imogen's response is what you would be aiming for, had they not already made it. Quickly and efficiently resolve outstanding questions / requests, and give absolutely no reason for any of the technical teams to engage further.